tests/rest/v1/accounts.spec.ts

```diff
@@ -168,6 +169,22 @@ describe("account", () => {
     });
   });
 
+  it("mutes by ID for 1s", () => {
+    return sessions.use(2, async ([alice, bob]) => {
+      let relationship = await alice.rest.v1.accounts
+        .$select(bob.id)
+        .mute({ duration: 1 });
+      expect(relationship.muting).toBe(true);
+
+      await sleep(5000);
```

Comment:
I set 5 seconds since when I tested with mastodon.social, it takes 2-3 seconds for the latest state applied. It might be able to be shorter but it may cause flaky tests. Do you think 5 seconds of sleep is acceptable? I can find only 1 second sleep in any test.